_pages/en_US/troubleshooting.md

```diff
@@ -5,47 +5,17 @@ title: Troubleshooting
 {% include toc title="Table of Contents" %}
 
 ## Unlaunch
-
 ### Unlaunch freezes at `MISMATCH IN FAT COPIES`?
 twlnf has a critical bug that doesn't properly update the entire NAND after modifying it, which causes certain homebrew (like the Unlaunch installer) to throw an error.
 
 While fixable, the method to do so isn't set in stone and largely varies between systems. One method is to delete any DSiWare installed via twlnf in the past, but it has also been reported that moving *all* DSiWare to the SD card and back to the system can help in some cases.
```

Comment:
Mentioned this on Discord too, should add something about that you need to update if you downgraded with TWLnf